gatling-core/src/main/scala/io/gatling/core/session/el/ElCompiler.scala

```diff
@@ -262,13 +262,30 @@ case object RandomUUID extends ElPart[UUID] {
   def apply(session: Session): Validation[UUID] = version4UUID().success
 }
 
+case object RandomInt extends ElPart[Int] {
+  def apply(session: Session): Validation[Int] = ThreadLocalRandom.current().nextInt().success
+}
+
+final case class RandomIntRange(min: Int, max: Int) extends ElPart[Int] {
+  def apply(session: Session): Validation[Int] = (min + ThreadLocalRandom.current().nextInt((max - min) + 1)).success
+}
+
+case object RandomLong extends ElPart[Long] {
+  def apply(session: Session): Validation[Long] = ThreadLocalRandom.current().nextLong().success
+}
+
+final case class RandomLongRange(min: Long, max: Long) extends ElPart[Long] {
+
+  def apply(session: Session): Validation[Long] = (min + ThreadLocalRandom.current().nextLong((max - min) + 1)).success
+}
+
 class ElParserException(string: String, msg: String) extends Exception(s"Failed to parse $string with error '$msg'")
 
 object ElCompiler extends StrictLogging {
 
   private val NameRegex = """[^.#{}()]+""".r
   private val DateFormatRegex = """[^#{}()]+""".r
-  private val NumberRegex = "\\d+".r
+  private val NumberRegex = """\d+""".r
```

Comment:
Need to add support negative numbers

Reply:
Indeed

Reply:
But then, please rename the existing regex into  PositiveNumberRegex to not break existing usage.

Reply:
Yep, I did it. But hadn't pushed yet